P202.Training.Test/UsersServiceTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using P202.Training.Domain;
+using P202.Training.Domain.Models;
+using Xunit;
+
+namespace P202.Training.Test
+{
+    public class UsersServiceTest
+    {
+        [Fact]
+        public void UsersService_CreateData_WithoutDatabase()
```

Comment:
If you inject the repository as a dependency, it shouldn't matter if you're using a database or not... you'd be mocking the repository methods and test just the domain layer logic

Reply:
Same goes for the rest of the tests. And also, all the tests should have at least one assertion